.buildkite/nightly.yml

```
@@ -0,0 +1,6 @@
+steps:
+  - label: 'nightly tests'
+    command:
+      - "./nightly-tests.sh"
```

Cool, and Buildkite command paths are relative to the repo root, not relative to the pipeline.yaml.
```suggestion
      - ".buildkite/nightly-tests.sh"
```